sys/net/sock/sock_util.c

```diff
@@ -116,30 +117,33 @@ static char* _find_pathstart(const char *url)
 
 int sock_urlsplit(const char *url, char *hostport, char *urlpath)
 {
+    assert(url != NULL);
```

Comment:
this addition should be reflected in the documentation as follows:

```
@param[in]   url         URL to split. Not Null.
[...]
@pre `url != NULL`
```

Reply:
Added